cmd/common.go

```diff
@@ -121,27 +121,60 @@ func listRoles(ctx *cli.Context, context string) error {
 	return writer.Err()
 }
 
-func listRoleTemplateBindings(ctx *cli.Context, b []RoleTemplateBinding) error {
-	writer := NewTableWriter([][]string{
+func listRoleTemplateBindings(ctx *cli.Context, writerConfig *TableWriterConfig, rtbs []RoleTemplateBinding) error {
```

Comment:
If I'm not wrong the `cli.Context` is not needed anymore.

Reply:
You're absolutely right.